xtask/src/test.rs

```diff
@@ -18,12 +18,10 @@ pub fn run_tests(shell: Shell, mut args: Arguments) -> anyhow::Result<()> {
     };
     let llvm_cov_nextest_flags: &[_] = if llvm_cov {
         &["llvm-cov", "--no-cfg-coverage", "--no-report", "nextest"]
+    } else if list {
+        &["nextest", "list"]
     } else {
-        if list {
-            &["nextest", "list"]
-        } else {
-            &["nextest", "run"]
-        }
+        &["nextest", "run"]
```

Comment:
nitpick(non-blocking): This change looks like something that should be a separate commit, or maybe even a PR. I'm guessing `clippy` complained at you about this?

Reply:
Yup it was `clippy`, I'll revert this

Reply:
You can even keep the change! It's good to apply `clippy` suggestions that fire for this code. 🙂 Just want to make sure individual commits have specific changes. Otherwise, it can be a hazard for source management operations like `git revert` if something needs to reverted.